MANIFEST.in

```diff
@@ -10,3 +10,4 @@ prune docs/_build
 
 global-exclude *.pyc
 global-exclude .ipynb_checkpoints/*
+prune scripts/*
```

Comment:
for now, i needed to get rid of the scripts to pass through tests. However, we might want to include these in the source distribution and set a `scripts` or  `entrypoint` in the setup.py

see: https://docs.python.org/3.4/distutils/setupscript.html#installing-scripts and http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point